+++
draft = false
title = "python3-exceptiongroup 1.2.2-2"
version = "1.2.2-2"
description = "Backport of PEP 654 (exception groups)"
date = "2024-11-02T17:30:40"
aliases = "/packages/221141"
categories = ['devel-extra']
upstreamurl = "http://pypi.python.org/pypi/exceptiongroup"
arch = "x86_64"
size = "30776"
usize = "150884"
sha1sum = "418c70dc988198540ec7827bb67d778439a53d11"
depends = "['python3']"
reverse_depends = "['python3-trio']"
+++
### Description: 
Backport of PEP 654 (exception groups)

### Files: 
* /usr/lib/python3.13/site-packages/exceptiongroup-1.2.2.dist-info/LICENSE
* /usr/lib/python3.13/site-packages/exceptiongroup-1.2.2.dist-info/METADATA
* /usr/lib/python3.13/site-packages/exceptiongroup-1.2.2.dist-info/RECORD
* /usr/lib/python3.13/site-packages/exceptiongroup-1.2.2.dist-info/WHEEL
* /usr/lib/python3.13/site-packages/exceptiongroup/py.typed
* /usr/lib/python3.13/site-packages/exceptiongroup/_catch.py
* /usr/lib/python3.13/site-packages/exceptiongroup/_exceptions.py
* /usr/lib/python3.13/site-packages/exceptiongroup/_formatting.py
* /usr/lib/python3.13/site-packages/exceptiongroup/_suppress.py
* /usr/lib/python3.13/site-packages/exceptiongroup/_version.py
* /usr/lib/python3.13/site-packages/exceptiongroup/__init__.py
* /usr/lib/python3.13/site-packages/exceptiongroup/__pycache__/_catch.cpython-313.opt-1.pyc
* /usr/lib/python3.13/site-packages/exceptiongroup/__pycache__/_catch.cpython-313.pyc
* /usr/lib/python3.13/site-packages/exceptiongroup/__pycache__/_exceptions.cpython-313.opt-1.pyc
* /usr/lib/python3.13/site-packages/exceptiongroup/__pycache__/_exceptions.cpython-313.pyc
* /usr/lib/python3.13/site-packages/exceptiongroup/__pycache__/_formatting.cpython-313.opt-1.pyc
* /usr/lib/python3.13/site-packages/exceptiongroup/__pycache__/_formatting.cpython-313.pyc
* /usr/lib/python3.13/site-packages/exceptiongroup/__pycache__/_suppress.cpython-313.opt-1.pyc
* /usr/lib/python3.13/site-packages/exceptiongroup/__pycache__/_suppress.cpython-313.pyc
* /usr/lib/python3.13/site-packages/exceptiongroup/__pycache__/_version.cpython-313.opt-1.pyc
* /usr/lib/python3.13/site-packages/exceptiongroup/__pycache__/_version.cpython-313.pyc
* /usr/lib/python3.13/site-packages/exceptiongroup/__pycache__/__init__.cpython-313.opt-1.pyc
* /usr/lib/python3.13/site-packages/exceptiongroup/__pycache__/__init__.cpython-313.pyc
* /usr/share/doc/python3-exceptiongroup-1.2.2/LICENSE
* /usr/share/doc/python3-exceptiongroup-1.2.2/README.rst
